tracex_project/extraction/logic/modules/module_time_extractor.py

```diff
@@ -106,7 +107,6 @@ def fix_end_dates(row):
 
         return df
 
-
     def __get_snippet(self, sentence_id):
```

Comment:
Wait, why is the function still there?

Reply:
I've overlooked it. thanks for pointing it out